packages/core/src/Site/index.ts

```diff
@@ -1678,58 +1675,35 @@ export class Site {
     this.variableProcessor.addUserDefinedVariableForAllSites('timestamp', time);
   }
 
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars, class-methods-use-this
-  async rebuildPagesBeingViewed(_currentPageViewed: string) {
-    throw new Error('Method not implemented.');
-  }
-
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars, class-methods-use-this
-  async removeAsset(_removedPageFilePaths: string | string[]) {
-    throw new Error('Method not implemented.');
-  }
-
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars, class-methods-use-this
-  async backgroundBuildNotViewedFiles(_arg0?: any, _arg1?: any) {
-    throw new Error('Method not implemented.');
-  }
-}
-
-/**
- * Below are functions that are not compatible with the ES6 class syntax.
- */
-
-/**
- * Build/copy assets that are specified in filePaths
- * @param filePaths a single path or an array of paths corresponding to the assets to build
- */
-Site.prototype.buildAsset = delay(Site.prototype._buildMultipleAssets as () => Bluebird<unknown>, 1000);
+  /**
+   * Build/copy assets that are specified in filePaths
+   * @param filePaths a single path or an array of paths corresponding to the assets to build
+   */
+  buildAsset = delay(this._buildMultipleAssets as () => Bluebird<unknown>, 1000);
```

Comment:
Might cause a type problem in the future when serveUtil.js is migrated to TypeScript, but should be ok for now.